ToDoList/Default.aspx

```diff
@@ -17,7 +17,7 @@
                 <table>
                     <tr>
                         <th>Description</th>
-                        <th>&nbsp;</th>
+                        <th>Priority</th>
```

Comment:
Don't you need a third `<th>` element here now that you have three columns in your table?